nutrition_labels/grant_tagger.py

```diff
@@ -319,6 +341,12 @@ def load_model(self, output_path):
             self.model = pickle.load(f)
         with open(os.path.join(output_path, "vectorizer.pickle"), "rb") as f:
             self.vectorizer = pickle.load(f)
+        try:
```

Comment:
apparently try/except for opening a file instead of checking if it exists is more pythonic https://blog.finxter.com/check-if-a-file-exists-in-python/

Reply:
good to know - I hadn't given it much thought. Worrying to think something could happen to my files between checking they exist and opening though!